trie/zk/util.go

```diff
@@ -70,37 +67,32 @@ func setNodeHash(n TreeNode, hash *zkt.Hash) {
 		node.hash = hash
 	case *LeafNode:
 		node.hash = hash
```

Comment:
Does `n` never be `EmptyNode` or `HashNode`?

Reply:
`n` could be an `EmptyNode` or a `HashNode`, but it doesn't do anything, so i erased it.